State currState
---
State[] succStates
float64[] costs
string[] actions
